nemo/collections/asr/parts/numba/rnnt_loss/utils/cpu_utils/cpu_rnnt.py

```diff
@@ -137,6 +138,29 @@ def setup_probs(
                     self.log_probs2[offset + 1] = log_probs[idx(t, u, labels[u])]
 
 
+class LogSoftmaxGradModification(Function):
```

Comment:
Isn't this function a duplicate of the one in rnnt_numpy.py?

Reply:
Yeah, for some weird reason im hitting circular dependency when I try to call the one from that class, so I duplicated here. I will be sending a new PR with enormous speed improvements for RNNT loss soon (maybe a few days) and I can refactor it out there.